lib/job.js

```diff
@@ -14,7 +14,13 @@ var Job = module.exports = function Job(args) {
   delete args.agenda;
 
   // Process args
-  args.priority = parsePriority(args.priority) || 0;
+
+  // allow priority overrides (e.g. `job.now('foo', { priority: 'low' })`)
```

Comment:
Ah right I see, I would use the job API directly and cascade the saves, similar to how to how it was done in https://github.com/agenda/agenda/pull/451/commits/336ff6445803606a6dc468a6f26c637145790adc (please forgive the variable names):

```js
var now = Date.now();
var one = jobs.create('fifo-priority').schedule(now).priority('high');
var two = jobs.create('fifo-priority').schedule(now + 100).priority('low');
var three = jobs.create('fifo-priority').schedule(now + 200).priority('high');

one.save(function () {
  two.save(function () {
    three.save(function () {
      jobs.start();
    });
  });
});
// check jobs ...
```